optaplanner-examples/src/main/java/org/optaplanner/examples/common/business/SolutionBusiness.java

```diff
@@ -217,24 +213,21 @@ public boolean isSolving() {
     }
 
     public void registerForBestSolutionChanges(final SolverAndPersistenceFrame solverAndPersistenceFrame) {
-        solver.addEventListener(new SolverEventListener<Solution_>() {
-            // Not called on the event thread
```

Comment:
Motivation: race conditions are tricky, so in concurrent code, it's important that it's clear which thread is handling what.